ui/src/app/views/workspace/component.ts

```diff
@@ -39,6 +39,31 @@ class NotebookDescriptionFilter implements StringFilter<Notebook> {
     return notebook.description.toLowerCase().indexOf(search) >= 0;
   }
 }
+/*
+* Sort comparators used by the cohort and notebook data tables to
+* determine the order that the cohorts loaded into client side memory
+* are displayed.
+*/
+class CohortNameComparator implements Comparator<Cohort> {
```

Comment:
What would you think of just having a common localeStringComparator used by most of our tables?

Reply:
Related: There is some duplication here. Could we have something like `HasName` and `HasDescription` interfaces and then get something like:
```typescript
class DescriptionComparator implements Comparator<HasDescription> {
  compare(a: HasDescription, b: HasDescription) {
    return a.description.localeCompare(b.description);
  }
}
```